disruption_py/settings/log_settings.py

```diff
@@ -114,4 +117,15 @@ def setup_logging(self):
                 diagnose=True,
             )
 
-        logger.debug(os.getenv("VIRTUAL_ENV"))
+        # header
+        package = "disruption_py"
+        commit = get_commit_hash()
+        logger.log(
+            "SUMMARY",
+            f"> {package} "
+            + f"v{importlib.metadata.version(package)} "
+            + f"# {commit} "
+            + f"/ {os.getenv('USER')}@{os.uname().nodename}",
+        )
+        logger.debug(f"https://github.com/MIT-PSFC/disruption-py/commit/{commit}")
```

Comment:
@amdecker we could populate this URL using the repository field from pyproject, but I could not find an easy way...

Reply:
Maybe you already saw this, but https://docs.python.org/3/library/tomllib.html was added in 3.11. When we drop 3.10, it could be worth looking into because then we could to something like 
```
with open("pyproject.toml", "rb") as f:
    data = tomllib.load(f)
repo = data["tool"]["poetry"]["repository"]
```